app/scripts/models/entry-model.js

```diff
@@ -670,7 +670,7 @@ const EntryModel = Backbone.Model.extend({
         _.forEach(ranking, rankingEntry => {
             if (this._getFieldString(rankingEntry.field).toLowerCase() !== '') {
                 const calculatedRank = Ranking.getStringRank(
-                    searchString,
+                    searchString.toLowerCase(),
```

Comment:
Let's pass `textLower` in `rankComparator`, instead of doing `toLowerCase` for every entry.

Reply:
Hmm... textLower doesn't reach the rankComparator. It is prepared here:

[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L285](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L285)

but the prepared filter including textLower is not persisted into this.filter, so it is lost again here:

[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L277](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L277)

Shouldn't the filter better be prepared here:
[https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L267](https://github.com/keeweb/keeweb/blob/f0ff26ebb4e40f4c8728230b585f052feb471078/app/scripts/models/app-model.js#L267)

Reply:
Hm, isn't it passed here? https://github.com/keeweb/keeweb/blob/develop/app/scripts/util/comparators.js#L16

Reply:
Read it again, I see. Yes, I think it makes sense to move it there

Reply:
Done